web/components/application/hacker/Hacker.sass

```diff
@@ -0,0 +1,27 @@
+@import ./pages/PageOne
+
+div.application
+  width: 100%
+  display: flex
+  justify-content: center
+
+  > div
+    padding-top: 56px
+    padding-bottom: 56px
```

Comment:
when #39 is merged / if you pull that branch, might want to class-ify some of these paddings and margins. Same goes for `display: flex` - can use `flex jc-center` - see if we can reduce this to a minimum

I'll just leave one comment for this file 🎉